galleries/users_explain/quick_start.py

```diff
@@ -111,6 +111,42 @@
 # Artists are drawn to the **canvas**.  Most Artists are tied to an Axes; such
 # an Artist cannot be shared by multiple Axes, or moved from one to another.
 #
+# .. _plot_types_quickstart:
+#
+# Plot types
+# ==========
+#
+# For an overview of the different types of plot you can create with Matplotlib,
+# see the :ref:`Plot types gallery <plot_types>`. If you don't find the plot
+# type you need, it may still be possible to create the visualization you want
+# by combining existing :ref:`Artists <users_artists>` and customizing plot
+# elements. For example, `~.matplotlib.axes.Axes.plot` returns a
+# :class:`~.lines.Line2D` object and `~.matplotlib.axes.Axes.scatter` returns a
+# :class:`~.collections.LineCollection`.
+from matplotlib.lines import Line2D
+from matplotlib.patches import Rectangle
+
+# Plot the data
+x = np.array([3, 4, 9, 8, 9, 8, 0, 8, 4, 8])
+fig, ax = plt.subplots()
+ax.plot(x, label='Data')
+
+# Computing the mean and standard deviation of the data
+mean = np.mean(x)
+std = np.std(x)
+
+# Adding a horizontal line for the mean, and a rectangle representing the
+# standard deviation of the data
+mean_line = Line2D([0, 10], [np.mean(x)]*2, color='red', label='Mean')
+std_patch = Rectangle([0, mean-std], 10, 2*std, alpha=0.1, label='$\sigma$')
```

Comment:
```suggestion
std_patch = Rectangle([0, mean-std], 10, 2*std, alpha=0.1, label=r'$\sigma$')
```
We have this in some of our user code for the "invalid escape sequence".  There may be a better way.